testkit/src/main/java/org/apache/calcite/test/SqlOperatorFixtureImpl.java

```diff
@@ -163,6 +163,7 @@ void forEachQueryValidateAndThen(StringAndPos expression,
       SqlValidator validator = factory.createValidator();
       SqlNode n = parseAndValidate(validator, sql);
       assertNotNull(n);
+      tester.checkFails(factory, sap, expectedError, true);
```

Comment:
the last parameter to this call should be "runtime" and not "true"

Reply:
IMO the line 160 checks the runtime is true, so this line use `true` is ok.